docs/beamrunner/beamrunner-2-install.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: docs
+title:  Installing IBM Streams Runner for Apache Beam
+navtitle: Installing
+description:  Installing IBM® Streams Runner for Apache Beam involves downloading and extracting the Streams Runner toolkit, configuring environment variables, and creating a credentials file for your Streaming Analytics service.
+weight:  10
+published: true
+tag: beam
+prev:
+  file: beamrunner-1-intro
+  title: Introduction
+next:
+  file: beamrunner-2a-using
+  title: Using
+---
+
+Installing IBM® Streams Runner for Apache Beam involves downloading and extracting the Streams Runner toolkit, configuring environment variables, and creating a credentials file for your Streaming Analytics service. You don't need to install IBM Streams to use Streams Runner.
+
+## Before you start
+
+A Red Hat Enterprise Linux 6 or 7 environment is recommended for submitting Apache Beam 2.0 applications to the Streaming Analytics service in Bluemix.
+
+## Creating a Streaming Analytics service on Bluemix
+
+Before you can download Streams Runner, you must have a Streaming Analytics service so that you can open the Streams Console. The following video demonstrates how to create the service.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zz0jqt61Xkg" frameborder="0" allowfullscreen></iframe>
+
+<br>To create a Streaming Analytics service:
+
+1. On the [Bluemix catalog](https://console.ng.bluemix.net/catalog/services/streaming-analytics/?cm_mc_uid=05407033353914938482142&amp;cm_mc_sid_50200000=) page, log in. If you don't yet have a Bluemix account, you can create one.

```

Comment:
What are the parameters in the in the URL? This shorter link also works for me `https://console.bluemix.net/catalog/services/streaming-analytics`.

Reply:
Besides, that link points to Streaming Analytics Service instead of the catalog page (https://console.bluemix.net/catalog/)